lib/cisco_node_utils/cmd_ref/itd_service.yaml

```diff
@@ -0,0 +1,120 @@
+# itd_service
+---
+_exclude: [N3k, N5k, N6k, N8k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd <name>/'
+  set_context: ["itd <name>"]
+
+access_list:
```

Comment:
I was following this:
https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/stp_global.yaml#L158

I originally started doing the way you asked here, but during channel_group development, Chris suggested to use "false" for default and ever since I changed it to that model. 
I can go back to my original thinking, but we need a standard way, we can discuss this tomorrow and go from there